lib/process-services/src/lib/process-list/components/start-process.component.html

```diff
@@ -4,35 +4,50 @@
         <div class="subtitle" id="error-message" *ngIf="errorMessageId">
             {{errorMessageId|translate}}
         </div>
-        <mat-form-field class="adf-process-input-container" [floatLabel]="'always'">
-            <mat-label>{{'ADF_PROCESS_LIST.START_PROCESS.FORM.LABEL.TYPE' | translate}}</mat-label>
-            <input
-                type="text"
-                matInput
-                [formControl]="processDefinitionInput"
-                [matAutocomplete]="auto"
-                id="processDefinitionName"
-                #inputAutocomplete>
-            <div class="adf-process-input-autocomplete">
-                <mat-autocomplete
-                    #auto="matAutocomplete"
-                    id="processDefinitionOptions"
-                    [displayWith]="displayFn">
-                    <mat-option *ngFor="let processDef of filteredProcesses | async" [value]="processDef.name"
-                    (click)="processDefinitionSelectionChanged(processDef)">
-                        {{ processDef.name }}
+        <div class="adf-start-process-definition-container">
+            <mat-form-field *ngIf="showSelectApplicationDropdown" [floatLabel]="'always'" class="adf-start-process-app-list">
+                <mat-select
+                    placeholder="{{'ADF_PROCESS_LIST.START_PROCESS.FORM.LABEL.APPLICATIONS' | translate}}"
+                    (selectionChange)="onAppSelectionChange($event)"
```

Comment:
I think we established a rule not to use this format (onSomething) some time ago. I remember @mauriziovitale was changing all the occurrences of this format to something else

Reply:
I guess the rule is applicable for the class properties
`onSomething: new EventEmitter<any>()`, 
                         TO
`something: new EventEmitter<any>()`, 

The catch block should have `on` as prefix  (something)="onSomething($event)". Correct me if am i wrong